LDM-533.tex

```diff
@@ -231,7 +236,17 @@ \subsection{Suspension criteria and resumption requirements}
 
 \subsection{Naming convention}
 
-All tests are named according to the pattern \textsc{prod-xx-yy} where:
+With the introduction of the JIRA Test Management, the following definition have to be considered:
+
+\begin{description}
+  \item[LVV]{: Is the label for the ``LSST Verification and Validation'' project in Jira where all information regarding tests are managed.}
+  \item[LVV-XXX]{: Are Verification Elements, where XXX is the Verification Element identifier.  Each Verification Element is derived from a requirement and has at least one Test Case associated. There can be multiple Verification Elements associated with a requirement (in Magic Draw).}
```

Comment:
“MagicDraw”, not “Magic Draw”. But actually, you probably don't need to mention it here at all: the reader of this document should not need to care about MagicDraw.

Reply:
(What would be really helpful is to have a description somewhere of what verification elements are conceptually. Why do we link VEs to requirements and test cases to VEs, rather than simply linking test cases to requirements? I don't think that explanation has to go in this document, but it has to end up somewhere — perhaps LDM-503.)

Reply:
Regarding this last point I have no answer. @tim pointed out a SPIE paper that has some information about it. In general it should be SE to provide these definitions.

Reply:
https://ls.st/Document-28669